onnxruntime/test/onnx/TestCase.cc

```diff
@@ -794,7 +794,7 @@ void LoadTests(const std::vector<std::basic_string<PATH_CHAR_TYPE>>& input_paths
       if ((reinterpret_cast<OnnxModelInfo*>(model_info.get()))->HasDomain(ONNX_NAMESPACE::AI_ONNX_TRAINING_DOMAIN) ||
           (reinterpret_cast<OnnxModelInfo*>(model_info.get()))->HasDomain(ONNX_NAMESPACE::AI_ONNX_PREVIEW_TRAINING_DOMAIN)) {
         fprintf(stderr, "Skip test case:: %s %s\n", ToUTF8String(test_case_name_in_log).c_str(), " as it has training domain");
-        return true;
+        // return true;
```

Comment:
The current HEAD is based off an old revision of Onnxruntime, which does not play well with int8 models, so they just don't support it. This return statement is the part of the logic that stops the execution on int8 models, and returns immediately.
That is why this return is commented out, and I will remove this altogether once we rebase on top of the latest HEAD of Onnxruntime, which I think should support them out of the box.

Reply:
This is because some models ship with a training domain, which we would like to compile through the EP, and that is a good number at the moment. Examples are DLV3_int8 and others. This should not be needed once we rebase on the latest commit in onnxrt's upstream main.

Reply:
This does not affect python bindings per se, only onnx_test_runner. Is you feel that this is not needed, I think it can be safely removed.